dumpling/export/status.go

```diff
@@ -49,7 +57,8 @@ type DumpStatus struct {
 	FinishedRows      float64
 	EstimateTotalRows float64
 	TotalTables       int64
-	CurrentSpeedBPS   int64
+	CurrentSpeedBPS   string
```

Comment:
I think numerical value contains more detailed information, we should not change it to string. For example, if frontend want to change the precision, it would be easier for a numerical value